pkg/uwb-core/Makefile.dep

```diff
@@ -19,7 +19,8 @@ endif
 ifneq (,$(filter uwb-core_dpl,$(USEMODULE)))
   USEPKG += mynewt-core
   USEMODULE += mynewt-core_os
-  ifneq (,$(filter nrf5%,$(CPU)))
+  # don't pull nrf53 into the list
+  ifneq (,$(filter nrf51 nrf52,$(CPU)))
     USEMODULE += mynewt-core_nrf5x_hal
```

Comment:
This one also has support for `nrf5340` and `nrf91xx` - but why does it use vendor specific code in the first place?

Reply:
I don't know. I never really used neither nimble nor mynewt.
I also don't understand why this didn't occur sooner...
I'll look at nimble support for nRF53 in RIOT at some point but let's focus on basic stuff first.